test/instrumentation/modules/http/basic.js

```diff
@@ -82,23 +82,41 @@ test('new http.Server', function (t) {
     server.on('request', onRequest(t))
     sendRequest(server)
   })
+
+  t.test('support elastic header', function (t) {
```

Comment:
I would probably name this test a little more descriptive than "elastic header" as it's not clear which header this refers to, eg:

```suggestion
  t.test('support elastic-apm-traceparent header', function (t) {
```